src/utils/layer-utils.js

```diff
@@ -99,10 +99,11 @@ export function getLayerHoverProp({
         return null;
       }
       const {dataContainer, fields} = datasets[dataId];
-      const data = layer.getHoverData(object, dataContainer, fields);
+      const data = layer.getHoverData(object, dataContainer);
```

Comment:
`fields` should be added to base-layer `getHoverData` method, instead of removing it here